### DISCUSSION TITLE
Map Screenshot ModHub

### DISCUSSION THREAD

**Discussion by BalticHawk** [Courseplay/Courseplay_FS22#1758] [Q&A] [answered]
Hey there 👋🏼
can you tell me what map you used for the last screenshot on ModHub?

![7A29DCB5-B172-4EE0-9850-97A53AAAECE3](https://user-images.githubusercontent.com/37582657/178158750-76bb7ee9-5edc-4678-8f88-9b7fdeade606.jpeg)

Looks very interesting for me. Or is this map private?

**Comment by Tensuko** [comment, marked as answer]
That is the "Landkreis Rostock" Map :)
A very nice map:
https://forbidden-mods.de/filebase/file/33000-landkreis-rostock/

**Reply by BalticHawk** [reply]
Thank you very much. 😃
